big_five_inventory_survey/config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Big Five Inventory Survey",
+    "exp_id": "big-five-survey",
+    "description": "A short form designed for researchers wanting to assess social desirability when time is limited.",
+    "instructions": "Here are a number of characteristics that may or may not apply to you.  For example, do you agree that you are someone who likes to spend time with others?  Please write a number next to each statement to indicate the extent to which you agree or disagree with that statement.",
+    "url": "https://github.com/expfactory-experiments/big-five-survey",
```

Comment:
Also I see you've added a URL here for expfactory-experiments - did you intend to contribute all of these there?

Reply:
Oops sorry, I forgot to delete "boredom_proneness_scale(short)". Only different thing is the title. You've added the correct one.

For the URL, I just followed the format written in other surveys. Guess config.json in expfactory-surveys is an old version. They don't have URL in it. So I referred to files in expfactory-experiments.